.github/workflows/govmomi-govc-tests.yaml

```diff
@@ -45,3 +45,51 @@ jobs:
 
       - name: Run govc Tests
         run: make govc-test
+
+  govc-tests-sso:
```

Comment:
I think we can simplify this workflow file into one job using matrix tests, e.g:

⚠️ please verify as I'm typing from my head :)

```yaml
jobs:
  govc-tests:
    name: Run govc Tests
    strategy:
      matrix:
        go-version: ["1.15", "1.16"]
        platform: ["ubuntu-18.04"]
        cmd: ["govc-test", "govc-test-sso-assert-cert"]
        experimental: [false]
        include:
          - go-version: "1.15"
            platform: "ubuntu-18.04"
            cmd: "govc-test-sso"
            experimental: true
          - go-version: "1.16"
            platform: "ubuntu-18.04"
            cmd: "govc-test-sso"
            experimental: true

    runs-on: ${{ matrix.platform }}
    continue-on-error: ${{ matrix.experimental }}
    timeout-minutes: 10

    steps:
      - name: Set up Go ${{ matrix.go-version }}
        uses: actions/setup-go@v2
        with:
          go-version: ${{ matrix.go-version }}
        id: go

      - name: Check out code
        uses: actions/checkout@v2

      - name: Run ${{ matrix.cmd }}
        run: make ${{ matrix.cmd }}
```